### PR TITLE
chore: add general and workspace-provider configuration documentation

### PR DESCRIPTION
**Pull request by thedadams**

### CODE REVIEW

**Review by cjellick — APPROVED** [review]

File: docs/docs/04-configuration/01-general.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
 # General
-Document any configuration that doesnt fit into model or auth providers. Ideally i'd like to see most if not all env vars we use in our render deployments documented. This doesn't need to be a hand holding guide, just give a decent idea of what each one does. 
\ No newline at end of file
+
+The Otto8 server is configured via environment variables. The following configuration is available:
```
Comment (drpebcak) [inline docs/docs/04-configuration/01-general.md:3]: It's a nit, but are we calling this `Otto` or `Otto8`? I've seen it called both ways now in the docs.

File: docs/docs/04-configuration/04-workspace-provider.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+#### `WORKSPACE_PROVIDER_DATA_HOME`
+
+Specify the directory where workspaces are nested. The default is `$XDG_CONFIG_HOME/otto8/workspace-provider`.
+
+### The s3 provider configuration
+
+#### `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`, `WORKSPACE_PROVIDER_S3_BUCKET`
+
+These variables configure the s3 or s3-compatible access and bucket to use with the workspace provider.
```
Comment (drpebcak) [inline docs/docs/04-configuration/04-workspace-provider.md]: This reads like a sentence fragment.

**Review by drpebcak — CHANGES_REQUESTED** [review]

File: docs/docs/04-configuration/01-general.md (hunk 1 above)
Comment (thedadams) [reply docs/docs/04-configuration/01-general.md:3]: I've done a quick search in the docs and I believe I have made the name consistently `Otto8`.

File: docs/docs/04-configuration/04-workspace-provider.md (hunk 2 above)
Comment (thedadams) [reply docs/docs/04-configuration/04-workspace-provider.md]: I reworded this.

**Review by iwilltry42 — APPROVED** [review]
LGTM - I like how you sneaked a tiny change in this docs PR :grin:

**Review by drpebcak — APPROVED** [review]

**Pull request merged**